recipe/meta.yaml

```diff
@@ -24,10 +24,14 @@ requirements:
     - python-dateutil
 
 test:
+  source_files:
+    - {{ environ['SRC_DIR'] }}
```

Comment:
Windows in failing with:

```
test/source_files paths must use / as the path delimiter on Windows
```

Is that something that must be fixed in `conda-build` or I am not using `source_files` correctly @msarahan.